src/topology/maps.lean

```diff
@@ -561,7 +577,12 @@ lemma closed_embedding_id : closed_embedding (@id α) :=
 
 lemma closed_embedding.comp {f : α → β} {g : β → γ}
   (hg : closed_embedding g) (hf : closed_embedding f) : closed_embedding (g ∘ f) :=
-⟨hg.1.comp hf.1, show is_closed (range (g ∘ f)),
- by rw [range_comp, ←hg.closed_iff_image_closed]; exact hf.2⟩
-
+⟨hg.to_embedding.comp hf.to_embedding, show is_closed (range (g ∘ f)),
+ by rw [range_comp, ←hg.closed_iff_image_closed]; exact hf.closed_range⟩
+
+lemma subtype_val.closed_embedding {s : set α} (hs : is_closed s) :
+  closed_embedding (subtype.val : {x // x ∈ s} → α) :=
+{ induced := rfl,
+  inj := subtype.val_injective,
+  closed_range := (subtype.val_range : range subtype.val = s).symm ▸ hs }
```

Comment:
missing empty line here, otherwise this looks good to me.

Reply:
Fixed (what a nice commit!).